index.html

```diff
@@ -44,73 +43,82 @@ <h1 class="item1 h1">I’m Kirubel. <br>Glad to see you!</h1>
     </ul>
   </section>
   <section class="work1">
-    <img class="work1-img" src="./images/Snapshoot Portfolio (1).png" alt="">
-    <h1>Tonic</h1>
+    <img class="work1-img" src="./images/port.png" alt="snapshot of website">
+    <p class="oldo">Tonic</p>
     <ul>
```

Comment:
- [ ] As mentioned by the previous reviewer, the margin between the headline and your works section is not as we have it on the Figma design. Kindly look into it and fix.

### Your Design
![your-header-projects](https://user-images.githubusercontent.com/26597432/227710533-0b16c487-4c2b-45b4-8b8b-f402def5a100.png)

### Figma Design
![temp-header-project](https://user-images.githubusercontent.com/26597432/227710546-8c4fb70b-8f98-44cd-a5bf-81dc84aa6b8e.png)